packages/app-desktop/gui/ResizableLayout/ResizableLayout.tsx

```diff
@@ -35,7 +32,7 @@ function itemVisible(item: LayoutItem, moveMode: boolean) {
 	return item.visible !== false;
 }
 
-function renderContainer(item: LayoutItem, parent: LayoutItem | null, sizes: LayoutItemSizes, onResizeStart: Function, onResize: Function, onResizeStop: Function, children: any[], isLastChild: boolean, moveMode: boolean): any {
+function renderContainer(item: LayoutItem, parent: LayoutItem | null, sizes: LayoutItemSizes, resizeMaxSize: Size | null, onResizeStart: Function, onResize: Function, onResizeStop: Function, children: any[], isLastChild: boolean, moveMode: boolean): any {
```

Comment:
As I understand resizeMaxSize is the max size of the items being currently resized, is that correct? If so, maybe make it more explicit `resizedItemMaxSize`, so that we know it's related to `resizedItem`.